Procfile

```diff
@@ -0,0 +1,2 @@
+web: env bin/rails server -p ${PORT:-3000}
+release: bin/rails db:prepare
```

Comment:
99% of the time, running db:prepare on an already created database simply runs migrations. But every so often I add some extra steps that should happen post-deploy, and I hook into the db:prepare command for that, so I'm leaning into prepare as the official release command in all hosted environments.